### DISCUSSION TITLE
Where can I found the latest lsposed framework api?

### DISCUSSION THREAD

**Discussion by Andy312432** [LSPosed/LSPosed#1635] [Q&A] [answered]
I have googled it , only to find to run actions in the old project: xposedbridge .
Where can I find the latest framework.Jar... or at least, manually generate it?

**Comment by mywalkb** [comment, marked as answer]
the latest framework is that released from rovo89.
For generate manually I used this method:
- build relase
- make a jar of LSPosed/tree/master/core/src/main/java/de/robv/android/xposed
- import in a project as compileonly

I don't know if there are better methods
